<?xml version="1.0"?>
<launch>
  ### THIS LAUNCH FILE WILL LAUNCH NODES ON THE LOCAL MACHINE AND REMOTE MACHINE (TX2)

  <machine name="tx2" address="tegra-ubuntu" env-loader="/home/nvidia/catkin_ws/src/agbot_0/env.sh" user="nvidia" default="true"/> 
  
    ### Steering Linear Actuatory
    <include file="$(find agbot_0)/launch/include/jrk.launch" />

    ###
    <include file="$(find agbot_0)/launch/include/teleop.launch" >
        <arg name="joy_dev"             default="/dev/input/js0" />
        <arg name="ackermann_command"   default="/ackermann"/>
        <arg name="twist_command"       default="/roboclaw/cmd_vel"/>
        <arg name="max_forward_vel"     default="2.0"/> # in m/s
        <arg name="max_reverse_vel"     default="2.0"/> # in m/s
        <arg name="max_steering_angle"  default="0.7853982"/> # in radians (45 deg)
        <arg name="joy_msgs"            default="/joy"/>
        <arg name="wheel_base"          default="1.18"/> # distance between front and back tires in meters
    </include>


</launch>
